libs/pid/src/PID.cpp

```diff
@@ -4,6 +4,7 @@
 #include <mutex>
 
 using namespace PID;
```

Comment:
This whole files namespacing needs cleaning up lol. partially my fault for giving questionable advice here. will be cleaned in #347